src/templates/finops-hub/modules/dataExplorer.bicep

```diff
@@ -26,91 +32,97 @@ var clusterName = replace('${take(safeHubName, 22)}', '--', '-')
 // Resources
 //==============================================================================
 
-resource cluster 'Microsoft.Kusto/clusters@2022-12-29' = {
+resource adxCluster 'Microsoft.Kusto/clusters@2023-05-02' = {
   name: clusterName
   location: location
   tags: tags
   sku: {
-    capacity: 1
-    name: 'string'
-    tier: 'string'
+    capacity: 2
+    name: 'Standard_E2ads_v5'
+    tier: 'Standard'
   }
-  identity: {
-    type: 'string'
-    userAssignedIdentities: {}
-  }
-  properties: {
-    acceptedAudiences: [
-      {
-        value: 'string'
-      }
-    ]
-    allowedFqdnList: [
-      'string'
-    ]
-    allowedIpRangeList: [
-      'string'
-    ]
-    enableAutoStop: true
-    enableDiskEncryption: false
-    enableDoubleEncryption: false
-    enablePurge: false
-    enableStreamingIngest: false
-    engineType: 'V3'
-    keyVaultProperties: {
-      keyName: keyVaultName
-      keyVaultUri: 'string'
-      keyVersion: 'string'
-      userIdentity: 'string'
-    }
-    languageExtensions: {
-      value: [
-        {
-          languageExtensionImageName: 'string'
-          languageExtensionName: 'string'
-        }
-      ]
-      value: [
-        {
-          languageExtensionImageName: 'string'
-          languageExtensionName: 'string'
-        }
-      ]
-    }
-    optimizedAutoscale: {
-      isEnabled: bool
-      maximum: int
-      minimum: int
-      version: int
-    }
-    publicIPType: 'string'
-    publicNetworkAccess: 'string'
-    restrictOutboundNetworkAccess: 'string'
-    trustedExternalTenants: [
-      {
-        value: 'string'
-      }
-    ]
-    virtualClusterGraduationProperties: 'string'
-    virtualNetworkConfiguration: {
-      dataManagementPublicIpId: 'string'
-      enginePublicIpId: 'string'
-      subnetId: 'string'
-    }
-  }
-  zones: [
-    'string'
-  ]
+  // identity: {
```

Comment:
Should we just remove these?

Reply:
Not yet. Need to get to a point where data is flowing in and queries and passing data out... then I will cleanup. Want to keep all the settings handy in case we need to add things like managed identities, etc.